aea/cli/run.py

```diff
@@ -233,6 +133,116 @@ def _setup_connection(
     return connection
 
 
+def _try_to_load_required_protocols(ctx: Context):
```

Comment:
This is still needed for the connections. Otherwise I would have moved it to the registry load step.